pkg/spiffs/include/spiffs_config.h

```diff
@@ -20,9 +20,6 @@
 #include <unistd.h>
 #include <stdint.h>
 
-#define ENABLE_DEBUG (0)
-#include <debug.h>
-
```

Comment:
I wanted to find a better solution, but if not defined here, we cannot use the SPIFFS_DBG* macros.
But it sucks...